Example/FinancialConnections Example/FinancialConnections Example/Playground/helpers/BannerHelper.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  BannerUtility.swift
+//  FinancialConnections Example
+//
+//  Created by Krisjanis Gaidis on 10/2/23.
+//
+
+import UIKit
+
+final class BannerHelper {
```

Comment:
note that this code is in the "Examples" directory which by default isn't very polished (as long as it works OK its fine)